tests/unit/providers/test_thegrid_provider.py

```
@@ -0,0 +1,83 @@
+import pytest
```

Ruff (0.16.3)

[convention] Missing copyright notice at top of file

(CPY001)